### PR TITLE
Eapi5 cleanup

### PR DESCRIPTION
**Pull request by jsmolic**

### CODE REVIEW

File: net-wireless/wepattack/wepattack-0.1.3-r5.ebuild [hunk 1]
```diff
@@ -22,14 +23,21 @@ DEPEND="
 "
 
 RDEPEND="${DEPEND}
-john? ( || ( app-crypt/johntheripper app-crypt/johntheripper-jumbo ) )"
+	john? ( || (
+		app-crypt/johntheripper
+		app-crypt/johntheripper-jumbo
+	)
+)"
```
Comment (SoapGentoo) [inline net-wireless/wepattack/wepattack-0.1.3-r5.ebuild:30]: one more indentation for the last two lines

File: net-wireless/wepattack/wepattack-0.1.3-r5.ebuild [hunk 2]
```diff
@@ -22,14 +23,21 @@ DEPEND="
...
+	"${FILESDIR}"/${P}-missed-string.h-warnings-fix.patch
+)
 
 src_prepare() {
-	epatch "${FILESDIR}"/${P}-filter-mac-address.patch
-	epatch "${FILESDIR}"/${P}-missed-string.h-warnings-fix.patch
-	chmod +x src/wlan
+	default
+
+	fperms +x src/wlan
```
Comment (SoapGentoo) [inline net-wireless/wepattack/wepattack-0.1.3-r5.ebuild:40]: ...does this work? `fperms` implicitly adds `${D}`

File: sys-apps/apmd/apmd-3.2.2_p14-r2.ebuild [hunk 3]
```diff
@@ -1,40 +1,48 @@
...
 		x11-libs/libICE
 		x11-libs/libXt
-		x11-libs/libXext )"
-DEPEND="${RDEPEND}
+		x11-libs/libXext
+	)"
+DEPEND="${RDEPEND}"
+BDEPEND="
 	sys-devel/libtool
 	virtual/os-headers"
```
Comment (SoapGentoo) [inline sys-apps/apmd/apmd-3.2.2_p14-r2.ebuild:38]: `virtual/os-headers` on `CBUILD`? :thinking:

File: sys-apps/spu-tools/spu-tools-2.3.0.136.ebuild [hunk 4]
```diff
@@ -18,18 +19,20 @@ KEYWORDS="ppc ppc64"
 RDEPEND="
 	sys-libs/ncurses:0=
```
Comment (SoapGentoo) [inline sys-apps/spu-tools/spu-tools-2.3.0.136.ebuild:20]:
```suggestion
	sys-libs/ncurses:=
```

File: sys-apps/spu-tools/spu-tools-2.3.0.136.ebuild [hunk 5]
```diff
@@ -18,18 +19,20 @@ KEYWORDS="ppc ppc64"
...
+	"${FILESDIR}"/${P}-format-security.patch
+)
 
 src_prepare() {
-	epatch \
-		"${FILESDIR}"/${P}-buildsystem.patch \
-		"${FILESDIR}"/${P}-format-security.patch
+	default
+
 	tc-export CC
```
Comment (SoapGentoo) [inline sys-apps/spu-tools/spu-tools-2.3.0.136.ebuild:36]: drop `src_prepare`, and do all fo this in `src_configure`

File: sys-apps/tomoyo-tools/tomoyo-tools-2.5.0_p20130214-r1.ebuild [hunk 6]
```diff
@@ -1,29 +1,31 @@
...
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 
-COMMON_DEPEND="sys-libs/ncurses:="
-DEPEND="${COMMON_DEPEND}
-	virtual/pkgconfig"
-RDEPEND="${COMMON_DEPEND}
+DEPEND="sys-libs/ncurses:="
+BDEPEND="virtual/pkgconfig"
```
Comment (SoapGentoo) [inline sys-apps/tomoyo-tools/tomoyo-tools-2.5.0_p20130214-r1.ebuild:19]: past `RDEPEND`

File: sys-apps/tomoyo-tools/tomoyo-tools-2.5.0_p20130214-r1.ebuild [hunk 7]
```diff
@@ -35,7 +37,7 @@ src_prepare() {
 		-e 's|-lncurses|$(shell ${PKG_CONFIG} --libs ncurses)|g' \
 		usr_sbin/Makefile || die
 
-	echo "CONFIG_PROTECT=\"/usr/$(get_libdir)/tomoyo/conf\"" > "${T}/50${PN}"
+	echo "CONFIG_PROTECT=\"/usr/$(get_libdir)/tomoyo/conf\"" > "${T}/50${PN}" || die
```
Comment (SoapGentoo) [inline sys-apps/tomoyo-tools/tomoyo-tools-2.5.0_p20130214-r1.ebuild:40]: do this in `src_install`, and do the pipe-straight-into-`newenvd`-idiom

File: sys-fs/jdiskreport-bin/jdiskreport-bin-1.4.1.ebuild [hunk 8]
```diff
@@ -16,12 +16,11 @@ SRC_URI="http://www.jgoodies.com/download/${MY_PN}/${MY_P}.zip"
 LICENSE="freedist"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
 
-DEPEND="app-arch/unzip"
+BDEPEND="app-arch/unzip"
```
Comment (SoapGentoo) [inline sys-fs/jdiskreport-bin/jdiskreport-bin-1.4.1.ebuild:20]: after `RDEPEND`

File: sys-fs/jdiskreport-bin/jdiskreport-bin-1.4.1.ebuild [hunk 9]
```diff
@@ -16,12 +16,11 @@ SRC_URI="http://www.jgoodies.com/download/${MY_PN}/${MY_P}.zip"
...
 RDEPEND=">=virtual/jre-1.6"
 
-S=${WORKDIR}/${MY_PN}-${PV}
+S="${WORKDIR}/${MY_PN}-${PV}"
```
Comment (SoapGentoo) [inline sys-fs/jdiskreport-bin/jdiskreport-bin-1.4.1.ebuild:23]: directly after `SRC_URI`?

File: x11-misc/gbdfed/gbdfed-1.6.ebuild [hunk 10]
```diff
@@ -11,19 +10,15 @@ SRC_URI="http://sofia.nmsu.edu/~mleisher/Software/gbdfed/${P}.tar.bz2"
...
+RDEPEND="
+	>=x11-libs/gtk+-2.6:2
 	>=media-libs/freetype-2
 	x11-libs/libX11
 	x11-libs/pango"
 DEPEND="${RDEPEND}"
 
 src_prepare() {
+	default
 	sed "s:-D.*_DISABLE_DEPRECATED::" -i Makefile.in || die #248562
```
Comment (SoapGentoo) [inline x11-misc/gbdfed/gbdfed-1.6.ebuild:23]: can't we just turn this into a patch?

File: x11-misc/lsw/lsw-0.3.ebuild [hunk 11]
```diff
@@ -13,10 +14,12 @@ SLOT="0"
 KEYWORDS="amd64 ~x86"
 
 RDEPEND="x11-libs/libX11"
-DEPEND="${RDEPEND}
-	x11-base/xorg-proto"
+DEPEND="${RDEPEND}"
+BDEPEND="x11-base/xorg-proto"
```
Comment (SoapGentoo) [inline x11-misc/lsw/lsw-0.3.ebuild:18]: sure? `xorg-proto` are just headers for the X data structures, i.e., it has to be for `CHOST`

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: x11-misc/gbdfed/gbdfed-1.6.ebuild [hunk 12]
```diff
@@ -11,19 +10,15 @@ SRC_URI="http://sofia.nmsu.edu/~mleisher/Software/gbdfed/${P}.tar.bz2"
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
 
-RDEPEND=">=x11-libs/gtk+-2.6:2
+RDEPEND="
+	>=x11-libs/gtk+-2.6:2
```
Comment (thesamesam) [inline x11-misc/gbdfed/gbdfed-1.6.ebuild:15]: Nit: sort

File: mail-filter/spampd/spampd-2.42.ebuild [hunk 13]
```diff
@@ -1,25 +1,25 @@
...
 SRC_URI="https://github.com/mpaperno/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~ppc x86"
-IUSE=""
 
-RDEPEND="dev-lang/perl
+RDEPEND="
+	dev-lang/perl
```
Comment (thesamesam) [inline mail-filter/spampd/spampd-2.42.ebuild:15]: fwiw usually Perl doesn’t need to be in DEPEND unless building e.g XS modules and would need c
:= in RDEPEND then anyway

But possible the configure script is checking for it so it’s a BDEPEND?

File: net-wireless/wepattack/wepattack-0.1.3-r5.ebuild (hunk 2 above)
Comment (jsmolic) [reply net-wireless/wepattack/wepattack-0.1.3-r5.ebuild:40]: oh you are right, we can't use it in `src_prepare`

File: mail-filter/spampd/spampd-2.42.ebuild (hunk 13 above)
Comment (jsmolic) [reply mail-filter/spampd/spampd-2.42.ebuild:15]: Hmm there isn't any sort of build system, it's just one perl script

File: net-wireless/wepattack/wepattack-0.1.3-r5.ebuild (hunk 1 above)
Comment (jsmolic) [reply net-wireless/wepattack/wepattack-0.1.3-r5.ebuild]: done

File: sys-apps/apmd/apmd-3.2.2_p14-r2.ebuild (hunk 3 above)
Comment (jsmolic) [reply sys-apps/apmd/apmd-3.2.2_p14-r2.ebuild:36]: :O

File: sys-apps/spu-tools/spu-tools-2.3.0.136.ebuild (hunk 5 above)
Comment (jsmolic) [reply sys-apps/spu-tools/spu-tools-2.3.0.136.ebuild:34]: done

File: sys-apps/tomoyo-tools/tomoyo-tools-2.5.0_p20130214-r1.ebuild (hunk 7 above)
Comment (jsmolic) [reply sys-apps/tomoyo-tools/tomoyo-tools-2.5.0_p20130214-r1.ebuild]: done

File: mail-filter/spampd/spampd-2.42.ebuild (hunk 13 above)
Comment (thesamesam) [reply mail-filter/spampd/spampd-2.42.ebuild:15]: Just RDEPEND then

**Review by thesamesam — APPROVED** [review]
lgtm, just make that Perl dep pure RDPEND

**Pull request merged**